engine/src/flutter/impeller/entity/contents/text_contents_unittests.cc

```diff
@@ -69,12 +69,12 @@ std::shared_ptr<GlyphAtlas> CreateGlyphAtlas(
 }
 
 Rect PerVertexDataPositionToRect(
-    GlyphAtlasPipeline::VertexShader::PerVertexData data[6]) {
+    GlyphAtlasPipeline::VertexShader::PerVertexData data[4]) {
```

Comment:
nit: Since we were refactoring these, it would have been nice to switch to std::array

Reply:
Done